opensearch-api/lib/opensearch/api/actions/cat/master.rb

```diff
@@ -23,7 +23,8 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
+        
+# <b>DEPRECATED:</b> Please use <tt>cluster_manager</tt> instead.
```

Comment:
Consider using YARD's `@deprecated` tag instead.

Reply:
Updated to follow this convention.